dingomata/cogs/poll.py

```diff
@@ -20,9 +21,12 @@ class PollUserError(DingomataUserError):
 
 
 class PollVoteButton(discord.ui.Button["PollVoteView"]):
-    def __init__(self, index: int):
+    def __init__(self, index: int, option_count: int):
         self.index = index
-        super(PollVoteButton, self).__init__(label=f"Vote {index + 1}", style=discord.ButtonStyle.blurple)
+        action_row = min(math.trunc(option_count / 5), 4)
+        super(PollVoteButton, self).__init__(label=f"Vote {index + 1}",
+                                             style=discord.ButtonStyle.blurple,
+                                             row=action_row)
```

Comment:
I think it's probably better to just take `row=` as an input here, and calculate it in the other function where you already have both variables. Oh, and instead of trunc, you can just use integer division like `a // b`

Reply:
Actually - iirc discord automatically splits components into rows so maybe we dont have to do this at all? :3c

Reply:
yeah I saw that it claimed to do and automatic split but it gave no details on how it was deciding on things or how many it would put on each row, so this was the more discrete option.  And now that I look at the way I wrong the decision equation its not going to work correctly anyhow.  Let me fix that